network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -306,6 +313,68 @@ def extract_topic_information(self, topic_slug):
             'topic': topic_object
         }
 
+    def get_related_topics(self):
+        related_topics = self.related_topics.all()
+        return related_topics
+
+    # Helper function to show if entry list is offset by topic box
+    def is_showing_topics_box_in_entry_list(self):
+        if (
+            not hasattr(self, "filtered")
+            and self.get_related_topics().count()
+            and (self.page_size == 12 or self.page_size == 24)
+        ):
+            return True
+        else:
+            return False
+
+    @route('^entries/')
```

Comment:
Per request from feedback, I have moved the function for rendering new entries here, and simplified a little bit by removing any unneeded logic + adding the check for whether or not we need to update the index by 1.